lib/asciidoctor/converter.rb

```diff
@@ -159,12 +159,20 @@ module Config
   #
   #   converter = Asciidoctor::Converter.create 'html5', htmlsyntax: 'xml'
   module Factory
-    # NOTE map deprecated initializer to initializer of CustomFactory
-    def self.new seed_registry = nil
-      CustomFactory.new seed_registry
+    # Public: Create a new instance of DefaultProxyFactory or CustomFactory depending on whether the proxy keyword arg
+    # is set (try by default), and optionally seed it with the specified converters map. If the proxy keyword arg is
```

Comment:
It was not previously an option. This is new. Though the default behavior matches how it worked before (since the default registry was actually scripted).